src/main/java/tr/com/srdc/cda2fhir/transform/entry/impl/EntryResult.java

```diff
@@ -7,47 +7,91 @@
 import org.hl7.fhir.dstu3.model.Bundle;
 import org.hl7.fhir.dstu3.model.Bundle.BundleEntryComponent;
 import org.hl7.fhir.dstu3.model.Resource;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.openhealthtools.mdht.uml.hl7.datatypes.CD;
 import org.openhealthtools.mdht.uml.hl7.datatypes.II;
 
+import tr.com.srdc.cda2fhir.transform.entry.CDAIIResourceMaps;
 import tr.com.srdc.cda2fhir.transform.entry.IEntityInfo;
 import tr.com.srdc.cda2fhir.transform.entry.IEntityResult;
 import tr.com.srdc.cda2fhir.transform.entry.IEntryResult;
 import tr.com.srdc.cda2fhir.transform.util.IDeferredReference;
+import tr.com.srdc.cda2fhir.transform.util.impl.CDACDMap;
 import tr.com.srdc.cda2fhir.transform.util.impl.CDAIIMap;
 import tr.com.srdc.cda2fhir.util.FHIRUtil;
 
 public class EntryResult implements IEntryResult {
-	private Bundle bundle;
-	private List<IDeferredReference> deferredReferences;
 
+	private Bundle newResourceBundle;
+	private Bundle fullBundle;
+	private List<IDeferredReference> deferredReferences;
```

Comment:
I would have prefered a new Class for this but it is OK for now.

Reply:
How would I go about that? Would the new class extend this one so that the entity map are available to it?

Reply:
Not sure right now. I think you should not need this fullBundle but no time to look deeper into it. good for now.